test/e2e/apps/disruption.go

```diff
@@ -189,10 +175,6 @@ var _ = SIGDescribe("DisruptionController", func() {
 			}
 
 			if c.shouldDeny {
-				// Since disruptionAllowed starts out false, wait at least 60s hoping that
-				// this gives the controller enough time to have truly set the status.
-				time.Sleep(timeout)
```

Comment:
won't this make the test flaky?

Reply:
I thought that the time was to allow the pod disruption budget to be processed.  The difference is that now we specifically wait for the pdb to be processed when we create or update the pdb in the waitForPdbToBeProcessed() function.

Reply:
> The difference is that now we specifically wait for the pdb to be processed when we create or update the pdb in the waitForPdbToBeProcessed() function.

That indicates the pdb controller has observed the update. It does not indicate the informer in the apiserver process (used by the eviction REST handler) has observed the update. In practice, the in-process informer is likely to always see the update events first, but that is not guaranteed.

Please keep an eye on this test post-merge to see if this increases flakiness.